dask/array/ghost.py

```diff
@@ -130,7 +130,7 @@ def ghost_internal(x, axes):
                  name, chunks)
 
 
-def trim_internal(x, axes=None):
+def trim_internal(x, axes):
```

Comment:
A lingering problem is that `trim_internal` cuts indiscriminately from both the inside and outside edges.  This can lead to odd behavior if `trim=True` and `boundary!=None`.